admin_app/src/app/page.tsx

```diff
@@ -18,21 +19,24 @@ export default function Home() {
   const [cardToEdit, setCardToEdit] = useState<Content | null>(null);
   const [newCardText, setNewCardText] = useState("");
 
-  const deleteCard = (id: string) => {
-    fetch(`${backendUrl}/content/${id}/delete`, {
-      method: "DELETE",
-      headers: get_api_headers(),
-    }).then((response) => {
-      if (response.ok) {
-        const newCardList = cards.filter(
-          (card: Content) => card.content_id !== id,
-        );
-        setCards(newCardList);
-        setFilteredCards(newCardList);
-      } else {
-        throw new Error("Could not delete " + id);
-      }
-    });
+  const deleteCard = (e: React.MouseEvent, id: string) => {
+    e.stopPropagation();
```

Comment:
This stop the page from reloading right? Is it needed when the button is not part of a form?

Reply:
Since the delete button is part of the card, if i click on delete without stop propogation, it also triggers the edit button. That is why it is there.